### PR TITLE
[quick7] update keyboard and keymap

### PR DESCRIPTION
**Pull request by MakotoKurauchi**
## Description
update keyboard and keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
> keyboard updates and refactors (eg. to data driven) must go through develop to reduce master -> develop merge conflicts

Also, all keyboard must have a default keymap so its deletion is incorrect.

File: keyboards/yushakobo/quick7/info.json [hunk 1]
```diff
@@ -1,55 +1,80 @@
...
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0002",
+        "vid": "0x3265"
+    },
+    "ws2812": {
+        "pin": "D3"
+    },
     "layouts": {
         "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:67]:
```suggestion
    "layout_aliases": {
        "LAYOUT": "LAYOUT_ortho_3x3"
    },
    "community_layouts": ["ortho_3x3"],
    "layouts": {
        "LAYOUT_ortho_3x3": {
```

File: keyboards/yushakobo/quick7/info.json [hunk 2]
```diff
@@ -1,55 +1,80 @@
...
+    "keyboard_name": "quick7",
     "maintainer": "yushakobo",
-    "usb": {
-        "vid": "0x3265",
-        "pid": "0x0002",
-        "device_version": "0.0.1"
+    "bootloader": "caterina",
+    "bootmagic": {
+        "enabled": true
     },
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:8]: already enabled in `features`
```suggestion
```

File: keyboards/yushakobo/quick7/info.json [hunk 3]
```diff
@@ -1,55 +1,80 @@
...
     "matrix_pins": {
         "direct": [
             ["D2", "D4", "F4"],
             ["D7", "B1", "B3"],
             ["E6", "B4", "B2"]
         ]
     },
+    "mouse_key": {
+        "enabled": false
+    },
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:37]: already enabled in `features`
```suggestion
```

File: keyboards/yushakobo/quick7/keymaps/default/rules.mk [hunk 4]
```diff
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/keymaps/default/rules.mk:1]:
```suggestion
ENCODER_MAP_ENABLE = yes

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/quick7/info.json [hunk 5]
```diff
@@ -1,55 +1,80 @@
...
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": true
     },
-    "ws2812": {
-        "pin": "D3"
-    },
-    "processor": "atmega32u4",
-    "bootloader": "caterina",
     "matrix_pins": {
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:28]:
```suggestion
    "development_board": "promicro",
    "matrix_pins": {
```

File: keyboards/yushakobo/quick7/info.json [hunk 6]
```diff
@@ -1,55 +1,80 @@
...
-    "keyboard_name": "quick7",
     "manufacturer": "yushakobo",
-    "url": "",
+    "keyboard_name": "quick7",
     "maintainer": "yushakobo",
-    "usb": {
-        "vid": "0x3265",
-        "pid": "0x0002",
-        "device_version": "0.0.1"
+    "bootloader": "caterina",
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:5]:
```suggestion
```

File: keyboards/yushakobo/quick7/info.json [hunk 7]
```diff
@@ -1,55 +1,80 @@
...
         "direct": [
             ["D2", "D4", "F4"],
             ["D7", "B1", "B3"],
             ["E6", "B4", "B2"]
         ]
     },
+    "mouse_key": {
+        "enabled": false
+    },
+    "processor": "atmega32u4",
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:38]:
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
replace `bootloader` and `processor` with `development_board`

File: keyboards/yushakobo/quick7/info.json [hunk 8]
```diff
@@ -1,55 +1,77 @@
...
-    "url": "",
+    "keyboard_name": "quick7",
     "maintainer": "yushakobo",
-    "usb": {
-        "vid": "0x3265",
-        "pid": "0x0002",
-        "device_version": "0.0.1"
-    },
     "encoder": {
+        "enabled": true,
```
Comment (dunk2k) [inline keyboards/yushakobo/quick7/info.json:6]: Already enabled in `features`
```suggestion
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/yushakobo/quick7/info.json [hunk 9]
```diff
@@ -1,55 +1,76 @@
...
-        }
-    },
-    "ws2812": {
-        "pin": "D3"
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": true,
+        "console": true,
```
Comment (zvecr) [inline keyboards/yushakobo/quick7/info.json:16]:
```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
```

**Pull request closed without merging**